activerecord/lib/active_record/persistence.rb

```diff
@@ -100,6 +100,10 @@ def persisted?
       !(@new_record || @destroyed)
     end
 
+    ##
+    # :call-seq:
+    #   save(*args)
```

Comment:
I tried using `:args:` to remove `&block` but didn't work at least in my local.
So I used `:call-seq:` instead, then it worked properly.

![image](https://cloud.githubusercontent.com/assets/12642/25215057/f3c4417c-25d5-11e7-8f97-44e194f2d50a.png)